code/modules/spacepods/spacepod.dm

```diff
@@ -343,15 +315,80 @@
 				if(!src || !WT.remove_fuel(3, user)) return
 				repair_damage(10)
 				to_chat(user, "\blue You mend some [pick("dents","bumps","damage")] with \the [WT]")
+			return
```

Comment:
Same thing here, although you can just remove this `return`, and remove one indent from the `return` in the `else`.